lib/readline.js

```diff
@@ -290,14 +291,14 @@ Interface.prototype.question = function(query, cb) {
 };
 
 
-Interface.prototype._onLine = function(line) {
+Interface.prototype._onLine = function(line, separator = '\n') {
```

Comment:
Do we need the default value here? Due to your changes it seems like the value is always set?

Reply:
@BridgeAR Hey thanks for taking time for the review. I've mostly passed around the appropriate values in the argument in the places I saw this function was being invoked. The idea behind the default value is to not alter its behavior if its called with falsy values by mistake by any call site.